game-core/src/main/java/games/strategy/engine/chat/ChatMessageListener.java

```diff
@@ -1,12 +1,9 @@
 package games.strategy.engine.chat;
 
 import games.strategy.engine.lobby.PlayerName;
-import java.util.Collection;
-
-/** An interface to allow for testing. */
-public interface IChatListener {
-  void updatePlayerList(Collection<ChatParticipant> players);
 
+/** Callback interface for a component that is interested in chat messages. */
+public interface ChatMessageListener {
```

Comment:
Don't most of our interfaces today start with "I"? Are we moving away from that standard? Just concerned about inconsistency.

Reply:
AFAIK we don't really use the I prefix for newer interfaces, but existing interfaces stay that way